### PR TITLE
Presets

### PR DESCRIPTION
**Pull request by patbenatar**
@taboularasa Allow for providing preset comments with labels. UI is ugly at the moment. I don't actually want to merge this into `v1.2.x`, so this PR is just a way for you to review the diff.

### CODE REVIEW

**Comment by patbenatar** [comment]
Given that `master` of this project is a total rewrite, I decided to take a concrete approach for our use case rather than an abstract pluggable style approach.

File: src/annotator.coffee [hunk 1]
```diff
@@ -59,9 +60,12 @@ class Annotator extends Delegator
   #     # Fallback for unsupported browsers.
   #
   # Returns a new instance of the Annotator.
-  constructor: (element, options) ->
+  constructor: (element, options={}) ->
     super
     @plugins = {}
+    @presets = @options.presets || []
+
+    this.subscribe 'annotationCreated', this.labelHighlight
```
Comment (taboularasa) [inline src/annotator.coffee:68]: why `this` instead of `@`

Comment (patbenatar) [inline src/annotator.coffee:68]: Lol this library has such inconsistent use of `this.` vs `@`, it was fucking with my head. I'll update all of my changes to use `@`

Comment (taboularasa) [inline src/annotator.coffee:68]: I don't think it matters much either way, I was just curious.

**Comment by taboularasa** [comment]
:shipit:

**Comment by patbenatar** [comment]
@taboularasa Thanks. Closing and we'll continue using the build off this branch.

**Pull request closed without merging**